src/components/prescriptions/ImportGroupComponent.vue

```diff
@@ -12,8 +12,8 @@
             <prescription-import-component
                 v-for="prescription in getPatientImportPrescriptions"
                 :key="prescription.id"
```

Comment:
github shows changes but I cannot spot it. Did u moved line below only?

Reply:
yes sir